examples/accesskey/accesskey.yaml

```diff
@@ -4,6 +4,11 @@ metadata:
   name: example
 spec:
   forProvider:
-    configurableField: test
+    repoName: test
+    projectName: test
```

Comment:
I think this also should be `projectKey`, as the bitbucket APIs refer to the project key and not the name directly.